eng/Common.Test.globalconfig

```diff
@@ -0,0 +1,1106 @@
+# Top level entry required to mark this as a global AnalyzerConfig file
```

Comment:
📝 All items should be removed from this file _except_ for items that are different from **Common.globalconfig**